src/workloads/execution/UnshardedLookup.yml

```diff
@@ -71,6 +95,38 @@ Actors:
             }}
           ]
         cursor: {batchSize: *batchSize}
+    - OperationMetricsName: MatchAllShardsLookupUnsharded
```

Comment:
Now that we have both sharded and unsharded to unsharded lookups, could you change this name to something like "ShardedToUnshardedLookupMatchAllShards", and same for the one below?

Oh and since I can't comment on unedited lines, similarly for line 70, "UnshardedToUnshardedLookupOneToFew"?